sources/api/schnauzer/src/helpers.rs

```diff
@@ -2080,6 +2117,86 @@ mod test_default {
     }
 }
 
+#[cfg(test)]
+mod test_if_not_null {
+    use super::*;
+    use handlebars::RenderError;
+    use serde::Serialize;
+    use serde_json::json;
+
+    // A thin wrapper around the handlebars render_template method that includes
+    // setup and registration of helpers
+    fn setup_and_render_template<T>(tmpl: &str, data: &T) -> Result<String, RenderError>
+    where
+        T: Serialize,
+    {
+        let mut registry = Handlebars::new();
+        registry.register_helper("if-not-null", Box::new(IfNotNullHelper));
+
+        registry.render_template(tmpl, data)
+    }
+
+    #[test]
+    fn null_value() {
+        let result = setup_and_render_template(
+            "{{#if-not-null setting}}foo{{/if-not-null}}",
+            &json!({"not-the-setting": 42.42}),
```

Comment:
Why is 42.42 the same as null?

Reply:
it's not, I'm just specifying another setting with a dummy value. I suppose this test would be the same if I just passed `{}` as the settings json, so maybe that's more readable. was just following the convention of other tests in this file